website_snippet_product_category/views/product_public_category.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="product_public_category_form_view" model="ir.ui.view">
+            <field name="name">public.category.form</field>
+            <field name="model">product.public.category</field>
+            <field
+                name="inherit_id"
+                ref="website_sale.product_public_category_form_view"
+            />
+            <field name="arch" type="xml">
+                <field name="image_1920" position="before">
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            class="oe_stat_button"
+                            name="website_publish_button"
+                            type="object"
+                            icon="fa-globe"
+                        >
+                            <field name="is_published" widget="website_button" />
```

Comment:
I can't see this button correctly in the runbot:
![image](https://user-images.githubusercontent.com/38977934/114505204-eedc3a80-9c27-11eb-8258-ed3764fcd734.png)